### DISCUSSION TITLE
Using `pinia-plugin-persistedstate` with `pinia-shared-state`

### DISCUSSION THREAD

**Discussion by rudolfbyker** [vuejs/pinia#2021] [Help and Questions]
Have any of you successfully used `pinia-plugin-persistedstate` together with `pinia-shared-state`? I'm wondering whether this will cause some kind of race condition, since there are multiple tabs, but only one local storage, and as far as I know, local storage is not reactive.

Use case:
- Many stores
- Some stores should be persisted (e.g. user's most recently used search terms)
- Some stores should be shared among tabs (I can't remember the specific example now)
- Some stores should be both persisted AND shared among tabs (e.g. user's preferences, like the theme)

**Comment by nongyehong** [comment]
Have you solved the problem,I seem to have the same problem
